### PR TITLE
E2e tests

### PR DESCRIPTION
**Pull request by dshaplyko**
My first pull request.
Created a structure of tests, page objects. Several e2e tests has been added.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -271,3 +271,11 @@ Oh, we are ashamed and want to fix it asap!
 But before fixing a bug we need to reproduce and confirm it. 
 
 In order to reproduce bugs we will systematically ask you to provide a minimal reproduce scenario (step by step).
+
+9. E2e testing
+---------------
+Testing is organized using Webdriver.io
+To execute test cases you need:
+
+1. npm run selenium-server
```
Comment (scolt) [inline readme.md]: You forgot to describe step with npm run e2e-setup (for installing drivers for chrome and FF).

File: tests/e2e/pageObjects/mainPage.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    //Elements
+
+
+    //Methods
+    navigateTo() {
+        return browser.url('/main');
+    }
+
+    waitForPageisLoaded() {
+        browser.waitUntil(function () {
```
Comment (scolt) [inline tests/e2e/pageObjects/mainPage.js]: Please use arrow functions in this case. 
Can be replaced with `browser.waitUntil( () => browser.getUrl() === (config.baseUrl + "/main"));`

File: tests/e2e/pageObjects/loginPage.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+export class LoginPage {
+
+    //Elements
+    welcomeText = "div.login-header";
+    emailField = "input[name='email']";
+    passwordField = "input[name='password']";
+    loginButton = "span[data-reactid='.0.1.0.1.2:$0.0.0.1.0']";
```
Comment (scolt) [inline tests/e2e/pageObjects/loginPage.js]: button[name="login"] looks better.

File: tests/e2e/pageObjects/loginPage.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    signUpButton = "div[data-reactid='.0.1.0.2.1']";
```
Comment (scolt) [inline tests/e2e/pageObjects/loginPage.js]: `.login-footer a` will be better for this case

File: tests/e2e/pageObjects/mainPage.js (hunk 2 above)
Comment (dshaplyko) [inline tests/e2e/pageObjects/mainPage.js]: It's done

File: readme.md (hunk 1 above)
Comment (dshaplyko) [inline readme.md]: it's done

File: tests/e2e/pageObjects/loginPage.js (hunk 4 above)
Comment (dshaplyko) [inline tests/e2e/pageObjects/loginPage.js]: Are you sure? Provided by you selector describes the whole "footer" of the menu, but I need only Sign Up button here.

File: tests/e2e/pageObjects/loginPage.js (hunk 3 above)
Comment (dshaplyko) [inline tests/e2e/pageObjects/loginPage.js]: it's done

File: tests/e2e/pageObjects/loginPage.js (hunk 4 above)
Comment (scolt) [inline tests/e2e/pageObjects/loginPage.js]: This selector describe only one link in the footer on login page. There are just one link in the footer SignUpButtom

Comment (dshaplyko) [inline tests/e2e/pageObjects/loginPage.js]: it's done

**Pull request merged**